src/containerapp/azext_containerapp/custom.py

```diff
@@ -2309,7 +2309,7 @@ def delete_java_component(cmd, java_component_name, environment_name, resource_g
     return java_component_decorator.delete()
 
 
-def create_java_component(cmd, java_component_name, environment_name, resource_group_name, target_java_component_type, configuration, service_bindings, unbind_service_bindings, no_wait):
+def create_java_component(cmd, java_component_name, environment_name, resource_group_name, target_java_component_type, configuration, service_bindings, unbind_service_bindings, min_replicas, max_replicas, no_wait):
```

Comment:
Are the new added argument required?

Reply:
It's not required. Can create the java component in new apiversion with old commands.

Reply:
Oh I got, `create_java_component` is not a function for command directly.